flow/scene_update_context.h

```diff
@@ -33,33 +36,34 @@ class SceneUpdateContext;
 
 #if defined(OS_FUCHSIA)
 
+using ContainerNode = mozart::client::ContainerTraits<mozart::client::Node>;
```

Comment:
Please add this to apps/mozart/lib/scene/client/resources.h instead of defining it here.  Then we can just say mozart::client::ContainerNode in this code.